### PR TITLE
Support for unshared services

### PR DESCRIPTION
**Pull request by judgedim**

### CODE REVIEW

File: tests/CreateTest.php [hunk 1]
```diff
@@ -249,6 +249,19 @@ public function services_can_be_marked_as_private()
...
      * @test
      */
+    public function services_can_be_marked_as_unshared()
+    {
+        $container = new ContainerBuilder;
+        (new PhpConfigLoader($container))->load([
+            'bar' => create('stdClass')
+                ->unshared()
+        ]);
+        self::assertFalse($container->findDefinition('bar')->isShared());
```
Comment (mnapoli) [inline tests/CreateTest.php]: Could you rather write a high level test? E.g. test that when you retrieve the service twice it is not the same instance.

That way the test is less coupled to the internals of the container.

Comment (judgedim) [reply tests/CreateTest.php]: sure

Comment (judgedim) [reply tests/CreateTest.php]: @mnapoli Added assert below

**Comment by mnapoli** [comment]
Awesome thanks!

One thing I missed (sorry I didn't spot that earlier) is that there should be a bit of documentation with the pull request too, could you add it?

**Comment by mnapoli** [comment]
Thanks!

**Pull request merged**